Formula/notcurses.rb

```diff
@@ -21,8 +21,13 @@ class Notcurses < Formula
   depends_on "libunistring"
   depends_on "ncurses"
   depends_on "readline"
+
   uses_from_macos "zlib"
 
+  on_linux do
+    depends_on "gcc"
+  end
```

Comment:
```suggestion
  end

  fails_with gcc: "5"
```

Reply:
it ought work with clang, fwiw

Reply:
Yea, but we normally fall back to GCC first before trying Clang. I'll just fix this in another PR.